src/pages/Flow/Run.vue

```diff
@@ -444,6 +465,54 @@ export default {
           </v-col>
         </v-row>
 
+        <v-row v-if="showAdvanced" class="my-2 py-8 row-divider" no-gutters>
+          <v-row class="my-2 py-8" no-gutters>
+            <v-col cols="12" md="3">
+              <div
+                class="py-0"
+                :class="{ 'pr-24': $vuetify.breakpoint.mdAndUp }"
+              >
+                <div class="text-h5">
+                  Logging Level
+                  <MenuTooltip>
+                    <p>
+                      Logging level for the flow run. If none is specified,
+                      default agent logging level will be used.
+                    </p>
+                    <p>
+                      This can also be controlled by providing an environment
+                      variable, "PREFECT__LOGGING__LEVEL".
+                    </p>
+                    <p>
+                      Any dropdown selection besides "Default" will override the
+                      environment variable.
+                    </p>
+                    <p>
+                      Please note this is only guaranteed to work for Agents
+                      running Prefect Core TODO or later.
+                    </p>
+                  </MenuTooltip>
+                </div>
+              </div>
+            </v-col>
+
+            <v-col cols="12" md="9" class="mt-n4 mt-md-0">
+              <v-select v-model="loggingLevel" outlined :items="loggingLevels">
```

Comment:
I'd add another template override here for the selected item for style continuity, something like this:
```
<template #selection="{ item }">
  <v-chip class="ma-2 debuglevel cursor-pointer" :class="item.text.toLowerCase()">
    {{ item.text}}
  </v-chip>
</template>
```
I'm not going to add this as a suggestion cause the formatting is likely off

Reply:
this looks so clean! i was wondering if there was an easy way to do this